lisp/ess-tracebug.el

```diff
@@ -56,11 +56,46 @@
 (require 'overlay)
 (eval-when-compile
   (require 'cl-lib))
+(require 'ess-custom)
 (require 'ess-utils)
 
 (defvar text-scale-mode-amount)
 (autoload 'text-scale-mode              "face-remap" "[autoload]" nil)
 
+;; Silence the byte compiler. This is OK here because this file is
+;; only loaded from ess-inf and has no autoloads.
+;; TODO: This is a LOT. Can we move some of this around?
```

Comment:
I think it should be the other way around. Here `(require 'ess-inf)` and `declare-function` in ess-inf.

Reply:
That makes sense to me, I'll fix it up.

Reply:
OK, done

Reply:
That's strange. I would guess sit's not much. Maybe it's worth creating a dedicated ess-r-inf and move tracebug there. In any case, we will have a clear picture of what is what once we remove all old files. let's not do any massive rearrangements till next release.

Reply:
OK, so I'll leave this as-is for now.